src/documents/regulations.html

```diff
@@ -1,142 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Huh... did I have 2x copies of all of these documents files? I guess so. That explains a few things.

Reply:
Yeah! 
I thought maybe they were an old version; didn't think much of it!